Core/Core.xcodeproj/project.pbxproj

```diff
@@ -4192,17 +4192,15 @@
 			files = (
 			);
 			inputFileListPaths = (
+				"$(SRCROOT)/../scripts/carthage/input.xcfilelist",
 			);
 			inputPaths = (
-				"$(SRCROOT)/../Carthage/Build/iOS/PSPDFKit.framework",
-				"$(SRCROOT)/../Carthage/Build/iOS/PSPDFKitUI.framework",
 			);
 			name = "Carthage Copy Frameworks";
 			outputFileListPaths = (
+				"$(SRCROOT)/../scripts/carthage/output.xcfilelist",
 			);
 			outputPaths = (
-				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/PSPDFKit.framework",
-				"$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/PSPDFKitUI.framework",
```

Comment:
wonder if this will help with our duplicate errors.

Reply:
I don't think so. I managed to track the duplicate thing down to a difference in HEADER_SEARCH_PATH in the binary pcm files, but I couldn't seem to make this change cause that.